src/Sandstorm/Camera.h

```diff
@@ -12,23 +12,14 @@
 #include "Camera.h"
 
 
-/*
-* Class for camera control
-*/
-class CameraControl {
-private:
-    Servo horizontalServo;
-    Servo verticalServo;
-    Servo armServo;
-    uint8_t horizontalMinAngle;
-    uint16_t horizontalMaxAngle;
-    uint8_t armHoldAngle;
-public:
-    CameraControl();
-    void cameraSetup(uint8_t horizontalPin, uint8_t verticalPin, uint8_t armPin);
-    void moveHorizontal(uint8_t horizontalAngle);
-    void moveVertical(uint8_t verticalAngle);
-    void moveArmHold(uint8_t armAngle);
-};
+extern Servo horizontalServo;
+extern Servo verticalServo;
+extern Servo armServo;
```

Comment:
I don't think this is best practice (having instantiated classes floating in the void of a header file). Instead, I would declare them in the main ino file and pass them as arguments. I'm not 100% sure on this though, I can look more into it.